database/seeders/FollowTableSeeder.php

```diff
@@ -15,6 +16,9 @@ class FollowTableSeeder extends Seeder
      * @return void
      */
     public function run() {
+        $gertrud = User::first();
```

Comment:
You don't want to get Getrud explicitly?

Reply:
Gertrud is always user number 1. Both in the code as well as in our hearts. 💕